scripts/build_clone_docs.sh

```diff
@@ -1,25 +1,60 @@
 # Clone the docs (draft or published)
 #
 # git clone does not like to clone into folders that are populated.  We are doing
-# this sequence of commands to workaround that limitation. 
+# this sequence of commands to workaround that limitation.
 # Could _not_ use:
 #   git clone git@github.com:kabanero-io/docs.git --branch develop src/main/content
 DOCS_GIT_URL=${1:-"https://github.com/kabanero-io/docs"}
 DOCS_GIT_REVISION=${2:-"master"}
 
 CUR_DIR="$(cd $(dirname $0) && pwd)"
+
+git_clone_doc_tags() {
+    #get all tags from the repo
+    TAGS=$(git ls-remote --tags $1 | cut -d/ -f3-)
+    #get the latest tag from repo
+    LATEST_TAG=$(git ls-remote --tags $DOCS_GIT_URL | awk -F'/' '/[0-9].[0-9].[0-9].*/ { print $3}' | sort -nr | head -n1)
+    
+    for TAG in $TAGS; do
+    #Remove the letter v or any other letter before the version number so VERSION only contains the numaric value of a release
+    VERSION=$(sed 's/[A-Za-z]*//g' <<<$TAG)
+
+    # If its the latest version of docs we want to clone it into the root of /docs
+    # for all other tags we clone them into a folder of thier own with the folder name being the version of doc under the /docs dir 
+        if [ "$TAG" == "$LATEST_TAG" ]; then
+            git clone -b $TAG --single-branch $DOCS_GIT_URL .
```

Comment:
you should only need to clone once and checkout the different versions to sub dirs maybe this will help https://stackoverflow.com/a/16493707/1309377 having clone in a loop seems a bit heavy to me